CMIP6_experiment_id.json

```diff
@@ -3761,6 +3790,35 @@
             ],
             "tier":"1"
         },
+        "hist-aer-cmip5":{
+            "activity_id":[
+                "DAMIP"
+            ],
+            "additional_allowed_model_components":[
+                "AER",
+                "CHEM",
+                "BGC"
+            ],
+            "description":"Historical anthropogenic-aerosols-only run (CMIP5-era forcing)",
```

Comment:
description `anthropogenic-aerosols-only` vs experiment `anthropogenic aerosols-only`

Reply:
For the CMIP6 counterpart it's `Historical anthropogenic-Aerosols-only run` which looks a bit funny. I don't know if we want to be consistent.

Reply:
@slavakharin yeah that was exactly what I noticed, I am inclined to tweak the original `hist-aer` while updating this, just to correct the weird formatting

Reply:
@durack1 while you at tweaking the description for these experiments, consider doing the same for `stratospheric-ozone-only` in `hist-stratO3` and `ssp245-stratO3`. Should it be `stratospheric ozone-only`? I am not sure about the correct grammar.

Reply:
@slavakharin, for transparency I may have to open a new issue to catch these subtle tweaks, this discussion being hidden in a PR is not as clearly obvious to watchers as it should be

Reply:
Resolved in #841 #842